modules/update-upgrading-oc-adm-upgrade-recommend.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/updating-cluster-cli.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="update-upgrading-oc-adm-upgrade-recommend_{context}"]
+= Finding recommended update paths with `oc adm upgrade recommend` (Technology Preview)
+
+When updating your cluster, it is useful to find out which versions are recommended for your update. While the `oc adm upgrade` command returns a list of the next available versions, this command decouples that information and narrows down your suggestions to recommend a new target release before you launch your update.
```

Comment:
Try to avoid subjective value/judgement statements, also not sure we need to repeat update in the forst line:
```suggestion
You can find out which versions are recommended for your update. While the `oc adm upgrade` command returns a list of the next available versions, this command decouples that information and narrows down your suggestions to recommend a new target release before you launch your update.
```

"this command" is a bit unclear - I think referring to the new `recommend` command? So maybe clarify the antecedent:
```suggestion
You can use the `oc adm upgrade recommend` command to see which versions are recommended for your update. While the `oc adm upgrade` command returns a list of the next available versions, this command decouples that information and narrows down your suggestions to recommend a new target release before you launch your update.
```

It still has a somewhat unclear antecedent. What about flipping it around a bit?
```suggestion
When updating your cluster, the `oc adm upgrade` command returns a list of the next available versions. You can use the `oc adm upgrade recommend` command to narrow down your suggestions and recommend a new target release before you launch your update.
```